mmdet/models/dense_heads/cascade_rpn_head.py

```diff
@@ -552,28 +555,31 @@ def _get_bboxes_single(self,
                            scale_factor,
                            cfg,
                            rescale=False):
-        """Transform outputs for a single batch item into bbox predictions.
+        """Transform outputs of single image into bbox predictions.
 
         Args:
-            cls_scores (list[Tensor]): Box scores for each scale level
-                Has shape (num_anchors * num_classes, H, W).
-            bbox_preds (list[Tensor]): Box energies / deltas for each scale
-                level with shape (num_anchors * 4, H, W).
-            mlvl_anchors (list[Tensor]): Box reference for each scale level
-                with shape (num_total_anchors, 4).
+            cls_scores (list[Tensor]):Box scores from all scale
+                levels of a single image, each item has shape
+                (num_anchors * num_classes, H, W).
```

Comment:
I believe we should distinguish the `num_anchors` and `num_base_anchors` which we did not pay attention before.
Here should be `num_base_anchors`

Reply:
Agree, num_base_anchors indicates the number of anchors in a point.